operators/pkg/controller/apmserverelasticsearchassociation/apmserverelasticsearchassociation_controller.go

```diff
@@ -189,13 +203,19 @@ func elasticsearchWatchName(assocKey types.NamespacedName) string {
 	return assocKey.Namespace + "-" + assocKey.Name + "-es-watch"
 }
 
+// esCAWatchName returns the name of the watch setup on Elasticsearch CA secret
```

Comment:
Technically this is not the CA secret but the public HTTP cert secret. Which we happen to use as a CA. I think we should make that clearer in that file.